### PR TITLE
Bugfix/pdct 664 update primary key constraint for physical document language table

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

Updated doc create DTO to have optional user_language_name.

Related linear tickets [PDCT-664](https://linear.app/climate-policy-radar/issue/PDCT-664/update-primary-key-constraint-for-physical-document-language-table) and [PDCT-666](https://linear.app/climate-policy-radar/issue/PDCT-666/language-should-not-be-mandatory).

## Type of change

Please select the option(s) below that are most relevant:

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/model/document.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class DocumentCreateDTO(BaseModel):
     # From PhysicalDocument
     title: str
     source_url: str
-    user_language_name: str
+    user_language_name: Optional[str]
```
Comment (diversemix) [inline app/model/document.py:59]: Nice driveby

Comment (katybaulch) [inline app/model/document.py:59]: This fixes https://linear.app/climate-policy-radar/issue/PDCT-666/language-should-not-be-mandatory
